test/unit/redirects_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+# Test existence of redirect routes
+class RedirectsTest < UnitTestCase
+  # Test existence of correct routes from MO old-style CRUDISH actions
+  # to normalized, resourceful Crud
+  def test_redirect_old_crud_actions
```

Comment:
Similar (but nonidentical) functionality to `test/integration/redirects_test.rb`
See discussion at https://github.com/MushroomObserver/mushroom-observer/pull/629/files?file-filters%5B%5D=.rb&hide-deleted-files=true#r468752266
I prefer this unit test to the integration test.